lib/sdk/storage.js

```diff
@@ -84,7 +96,23 @@ function extension(filename, mimeType) {
     return ext;
 }
 
-// There is at least one worker that in some cases depends
+/**
+ * Returns if the file exists and is not empty
+ *
+ * @param {Number} path location of the file
+ * @returns {Boolean} Returns false if file does not exist or is empty
+ */
+function fileExistsAndIsNotEmpty(path) {
+    if(path && fs.existsSync(path)){
```

Comment:
Why make it sync? Is it used in sync functions?

Reply:
This func is directly port from sdk. I believe it can be async, but I was told it is not a good idea to change things in sdk folder unless necessary.

Reply:
ACK